### PR TITLE
refactor: rename ServiceComponentLog to ComponentVersionLog

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Part of #348 

#### Additional comments

Rename the `service_component_log` table to `component_version_log` and rename related variables accordingly.

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp/cli/commands/browse.py [hunk 1]
```diff
@@ -134,16 +134,16 @@ def print_formatted_deployment(deployment_log):
 
 def print_formatted_operation_log(operation_log):
     headers = OperationLog.__table__.columns.keys()
-    service_headers = ServiceComponentLog.__table__.columns.keys()
+    service_headers = ComponentVersionLog.__table__.columns.keys()
     # TODO: this outputs Service and ServiceComponent version when it should
```
Comment (rpignolet) [inline tdp/cli/commands/browse.py:138]: This comment use "ServiceComponent" which should be updated no ?

File: tdp/cli/queries.py [hunk 2]
```diff
@@ -8,10 +8,10 @@
 from sqlalchemy.orm.session import Session
 from sqlalchemy.orm.exc import NoResultFound
 
-from tdp.core.models import DeploymentLog, ServiceComponentLog, OperationLog
+from tdp.core.models import DeploymentLog, ComponentVersionLog, OperationLog
 
 
-def get_latest_success_service_component_version_query(session: Session):
+def get_latest_success_component_version_log(session: Session):
     """Get the latest success service component version.
```
Comment (rpignolet) [inline tdp/cli/queries.py:15]:
```suggestion
    """Get the latest success component version.
```

File: tdp/cli/commands/browse.py (hunk 1 above)
Comment (PaulFarault) [reply tdp/cli/commands/browse.py]: Done

File: tdp/cli/queries.py (hunk 2 above)
Comment (PaulFarault) [reply tdp/cli/queries.py]: Done

**Comment by PaulFarault** [comment]
Force pushed after rebase on master.

**Review by rpignolet — APPROVED** [review]

File: tdp/cli/commands/browse.py [hunk 3]
```diff
@@ -107,13 +107,13 @@ def print_formatted_deployment(deployment_log):
             colalign=("right",),
         )
     )
-    if deployment_log.service_components:
+    if deployment_log.component_version:
         click.echo(
             "\nService Component logs:\n"
```
Comment (mdrutel) [inline tdp/cli/commands/browse.py:112]: Should we have ```Component version``` ?

File: tdp/core/deployment/deployment_plan.py [hunk 4]
```diff
@@ -134,14 +134,14 @@ def from_operations(operations: List[Operation]) -> "DeploymentPlan":
     def from_reconfigure(
         dag: Dag,
         cluster_variables: ClusterVariables,
-        service_component_deployed_version: ServiceComponentLog,
+        component_version_deployed: ComponentVersionLog,
     ) -> "DeploymentPlan":
         """Generate a deployment plan from a list of service component deployed version.
```
Comment (mdrutel) [inline tdp/core/deployment/deployment_plan.py:139]: Should we fix this comment ?

File: tdp/core/models/component_version_log.py [hunk 5]
```diff
@@ -9,10 +9,8 @@
...
 
-class ServiceComponentLog(Base):
-    """Component log model.
-
-    Hold what component version is deployed.
+class ComponentVersionLog(Base):
+    """Hold what component version are deployed.
 
     Attributes:
         id (int): Service component log id.
```
Comment (mdrutel) [inline tdp/core/models/component_version_log.py:16]: Should we fix this comment ?

File: tdp/cli/commands/browse.py (hunk 3 above)
Comment (PaulFarault) [reply tdp/cli/commands/browse.py]: Done

File: tdp/core/deployment/deployment_plan.py (hunk 4 above)
Comment (PaulFarault) [reply tdp/core/deployment/deployment_plan.py]: Done

File: tdp/core/models/component_version_log.py (hunk 5 above)
Comment (PaulFarault) [reply tdp/core/models/component_version_log.py]: Done

**Review by mdrutel — APPROVED** [review]

**Pull request merged**